services/log/log.go

```diff
@@ -1,25 +1,33 @@
 package log
 
 import (
-	"os"
-
 	"github.com/getconversio/go-utils/util"
-	"github.com/getconversio/logentrus"
+	"github.com/puddingfactory/logentrus"
 	log "github.com/sirupsen/logrus"
 )
 
-func Setup() {
+// Setup options for Open Exchange Rates
```

Comment:
You mean "for Logging" or something like that?

Reply:
hehe, it's copy-paste from oxr.go sorry :)